pkg/pmem-device-manager/pmd-ndctl.go

```diff
@@ -90,11 +90,12 @@ func (pmem *pmemNdctl) CreateDevice(name string, size uint64, nsmode string) err
 		glog.V(4).Infof("Device with name: %s already exists, refuse to create another", name)
 		return fmt.Errorf("CreateDevice: Failed: namespace with that name exists")
 	}
-	// align up by 1 GB, also compensate for libndctl giving us 1 GB less than we ask
+	// align by 1 GB
```

Comment:
Aligning implies that we ensure that the value is an integer multiple of some base value. Here we just enforce a minimum size. I would describe this as:
```
// Increase to minimum size supported by libndctl.
const minsize uint64 = 2 * 1024 * 1024 * 1024
if size < minsize {
    size = minsize
}
```

Reply:
you are right, I was lazy and re-used align value as minsizize value, will clean this up.